src/common/utils/isValidBusinessId.ts

```diff
@@ -1,7 +1,7 @@
 import { zipWith } from 'lodash';
 
 export default function isValidBusinessId(value: string | null | undefined): boolean {
-  if (value === null || value === '') return true;
+  if (value === null || value === '') return false;
```

Comment:
I think that for null or empty value this should return true because there is separate validation for required business ID.